docs/advanced-usage.md

```diff
@@ -261,10 +261,14 @@ steps:
   with:
     node-version: '14'
     cache: 'pnpm'
-- run: pnpm install --frozen-lockfile
```

Comment:
**Nit:** I'd suggest don't remove `--frozen-lockfile` from this block, as customers with pnpm lower than 6.10.x may have troubles. I believe only informational note below should be enough.

Reply:
I think it won't be the case because we support pnpm starting from 6.10+